### PR TITLE
refactor(ink): Mark APIs as `@alpha`

### PR DESCRIPTION
**Pull request by Josmithr**
The package has always been "experimental", and was never made ready for public/production use. The package itself predates our `@fluid-experimental` namespace, so it has continued to live in our (now) production namespace `@fluidframework`.

This PR moves the dds code into our "experimental" directory, and annotates its APIs as `@alpha`. It also updates the README to make the experimental nature a bit more clear.

Unfortunately, our test tooling relies on being able to download old versions of this package in an automated fashion based on its package name, so moving the package to the `@fluid-experimental` namespace is not a trivial change. We likely will want to do this in the future, but it can happen separately.

### CODE REVIEW

**Review by ChumpChief — CHANGES_REQUESTED** [review]
Let's not move it into experimental/ (that directory needs to go away) but flagging as alpha seems good.

**Comment by Josmithr** [comment]
> Let's not move it into experimental/ (that directory needs to go away) but flagging as alpha seems good.

Is that a plan of record? We just lean on package namespaces to differentiate? Happy to make the change, just want to make sure that organization change is explicitly planned work :)

**Comment by ChumpChief** [comment]
> Is that a plan of record? We just lean on package namespaces to differentiate? Happy to make the change, just want to make sure that organization change is explicitly planned work :)

In theory I think `@alpha` replaces both namespacing and the directory eventually.  I've personally been slowly moving packages out when I get the chance.

**Comment by Josmithr** [comment]
> > Is that a plan of record? We just lean on package namespaces to differentiate? Happy to make the change, just want to make sure that organization change is explicitly planned work :)
> 
> In theory I think `@alpha` replaces both namespacing and the directory eventually. I've personally been slowly moving packages out when I get the chance.

I don't think I agree with that wholesale. Namespaces are a good opportunity for us to communicate intentions with someone who may be looking at one of our packages on github or npm. If a package is marked as experimental, that signals a lack of readiness at the package level, rather than on an api-by-api level. And I think we definitely need our internal namespace (which we will need to document as "not intended for external use".

Even if _all_ of the APIs in the above packages are non-public, I think we need to make sure the intentions behind which packages are actually meant for external consumption / production use is clear.

File: packages/dds/ink/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	"repository": {
 		"type": "git",
 		"url": "https://github.com/microsoft/FluidFramework.git",
-		"directory": "packages/dds/ink"
+		"directory": "experimental/dds/ink"
```
Comment (ChumpChief) [inline packages/dds/ink/package.json:9]: Also needs revert
```suggestion
		"directory": "packages/dds/ink"
```

File: packages/dds/ink/package.json
Comment (ChumpChief) [inline packages/dds/ink/package.json:1]: Might also be reasonable to disable typeValidation for this package given alpha tag

**Review by ChumpChief — APPROVED** [review]

File: packages/dds/ink/package.json
Comment (Josmithr) [reply packages/dds/ink/package.json:1]: We're independently investigating updating the tooling to correctly omit (or not) `alpha`/`beta`/`internal` types in our type-tests. AB#5708. I'll leave any changes here for the results of that item.

**Comment by connorskees** [comment]
I am still of the opinion that the ink DDS should be removed, especially with potential upcoming DDS-wide changes to serialization, having a DDS that we don't use or intend on supporting in the future is an unnecessary maintenance burden.

**Comment by Josmithr** [comment]
Closing this for now, as we have changed our plans for how to approach migrating tags.

**Pull request closed without merging**